.circleci/config.yml

```diff
@@ -237,15 +245,14 @@ jobs:
       - capture_test_logs
 
   acceptanceTestsQuorum:
-    parallelism: 1
-    executor: xl_machine_executor
+    executor: quorum_ats_executor_xl
     steps:
       - prepare
       - attach_workspace:
           at: ~/project
       - run:
           name: Quorum Acceptance Tests
-          no_output_timeout: 40m
+          no_output_timeout: 30m
```

Comment:
I'm not sure if I'm understanding this correctly but it seems like we're changing the step from parallel to sequential but reducing the timeout? Shouldn't the timeout be longer (although 30m seems quite excessive)

Reply:
Fair point but this timeout is only used if no output is received for that length of time. This only happens if something goes seriously wrong.